src/SDKs/ManagedServices/ManagedServices.Test/ManagedServices.Test.csproj

```diff
@@ -0,0 +1,40 @@
+<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0">
+  <!-- Please do not move/edit code below this line -->
+  <Import Condition=" Exists('$([MSBuild]::GetPathOfFileAbove(AzSdk.test.reference.props))') " Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.test.reference.props'))" />
+  <!-- Please do not move/edit code below this line -->
+  <PropertyGroup>
+    <PackageId>ManagedServices.Test</PackageId>
+    <VersionPrefix>1.0.0.0</VersionPrefix>
+    <AssemblyName>ManagedServices.Test</AssemblyName>
+    <Description>ManagedServices.Test class library</Description>
+  </PropertyGroup>
+  <PropertyGroup>
+    <TargetFrameworks>netcoreapp2.0</TargetFrameworks>
+  </PropertyGroup>
+  <ItemGroup>
```

Comment:
Instead of picking individual json files, select all *.json files under the directory, this way new test recordings won't be missed

Reply:
good idea, thanks